### ISSUE TITLE
Timescale doesn't work when upgrading to Postgres 14

### ISSUE THREAD

**Issue by StaNov** [zalando/postgres-operator#1771]
I am trying to upgrade PG with Timescale from 13 to 14.

I get error in log:

```
FATAL:  extension "timescaledb" must be preloaded
HINT:  Please preload the timescaledb library via shared_preload_libraries.

This can be done by editing the config file at: /home/postgres/pgdata/pgroot/data/postgresql.conf
and adding 'timescaledb' to the list in the shared_preload_libraries config.
	# Modify postgresql.conf:
	shared_preload_libraries = 'timescaledb'

Another way to do this, if not preloading other libraries, is with the command:
	echo "shared_preload_libraries = 'timescaledb'" >> /home/postgres/pgdata/pgroot/data/postgresql.conf 

(Will require a database restart.)

If you REALLY know what you are doing and would like to load the library without preloading, you can disable this check with: 
	SET timescaledb.allow_install_without_preload = 'on';
server closed the connection unexpectedly
	This probably means the server terminated abnormally
	before or while processing the request.
```

Is there a way how to set the `postgres.conf` file from the manifest?

It seems that Patroni has parameter `custom_conf` which unfortunately cannot be passed from PGO manifest.

Thank you for your work!

---

Please, answer some short questions which should help us to understand your problem / question better?

- **Which image of the operator are you using?** registry.opensource.zalan.do/acid/postgres-operator:v1.7.1
- **Where do you run it - cloud or metal? Kubernetes or OpenShift?** Kubernetes, Google Cloud
- **Are you running Postgres Operator in production?** yes, but testing the upgrade elsewhere
- **Type of issue?** Question / Bug report

**Comment by shrumm** [comment]
Have a look at : https://github.com/zalando/postgres-operator/issues/1752

**Comment by StaNov** [comment]
Thank you, I'll try the upstream version.

Btw, is there any page with the available version list? I used the `spilo-14:2.1-p3` version because it is marked as the latest release.
